app/components/card-pipeline/stage-card.js

```diff
@@ -21,8 +21,10 @@ const PipelineCardComponent = Component.extend({
     return (this.get('stages') || []).filter(stage => get(stage, '_id') !== stageId);
   }),
 
-  nextMeeting: computed('application.[]', function () {
-    return this.get('getNextMeeting').perform();
+  nextMeeting: computed('application.[]', 'newScheduledMeeting', function () {
+    if (!this.meetingFetched || this.application.id !== this.newScheduleMeeting) {
```

Comment:
Are you sure this is working?
```suggestion
    if (!this.meetingFetched || this.application.id !== this.newScheduledMeeting) {
```

Reply:
yeah it works just fine, it just calls the query everytime no matter what :) NAI

Reply:
fixed